packages/react/src/primitives/shared/__tests__/styleUtils.test.tsx

```diff
@@ -149,6 +149,7 @@ describe('convertStylePropsToStyleObj: ', () => {
     const props: ViewProps = {
       color: 'red',
       fontWeight: 'bold',
+      padding: 'large',
```

Comment:
nit: it seems we don't test the `padding` in this single test. Could simply remove it or add one more `expect` below